### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by jdbohrman**
Let's try this again. Update to #14326

### CODE REVIEW

File: content/en/docs/setup/independent/install-kubeadm.md [hunk 1]
```diff
@@ -147,9 +147,12 @@ For more information on version skews, see:
 
 {{< tabs name="k8s_install" >}}
 {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+
+Commands must be run as [superuser](http://manpages.ubuntu.com/manpages/xenial/man8/sudo_root.8.html).
```
Comment (neolit123) [inline content/en/docs/setup/independent/install-kubeadm.md]: i'd update this like so, to be:
- distro agnostic
- to have the sentence in a "note block"

```
{{< note >}}
All commands must be run as a super user.
{{< /note >}}
```

File: content/en/docs/setup/independent/install-kubeadm.md [hunk 2]
```diff
@@ -147,9 +147,12 @@ For more information on version skews, see:
...
+
 ```bash
 apt-get update && apt-get install -y apt-transport-https curl
-curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
```
Comment (neolit123) [inline content/en/docs/setup/independent/install-kubeadm.md]: i would remove the `sudo` here given the above note.

**Comment by DanyC97** [comment]
@jdbohrman would be nice if you can rename the PR and provide a better title which describe what is for ? my 0.02$  thank you

**Comment by neolit123** [comment]
/check-cla

if the CLA command about does not remove the `cncf-cla: no` label, please double check your CLA registration with the CNCF helpdesk.
thanks.

**Comment by cody-clark** [comment]
Hi @otzoran . Thanks for your PR! Please sign the CLA, so we can start getting it merged.

> 📝 Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.

The contact info for the CNCF helpdesk is available in the [troubleshooting section](https://git.k8s.io/community/CLA.md#the-contributor-license-agreement)

**Comment by tengqm** [comment]
Emm ... "commit missing GitHub user", that is not a state for CLA checking. Feel free to reopen this when CLA is signed.

**Pull request closed without merging**